ex/app/src/main/java/com/example/ex/MainFragment.java

```diff
@@ -0,0 +1,209 @@
+package com.example.ex;
+
+import android.content.Context;
+import android.os.Bundle;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
```

Comment:
There was a certain condition regarding int ranges of numeric params in the task. Something about 1-6. And in the toast I can see 0-5 values.